subworkflows/yaml-schema.json

```diff
@@ -51,32 +51,51 @@
                     ".*": {
                         "type": "object",
                         "properties": {
-                            "type": {
-                                "type": "string",
-                                "description": "Type of the input channel"
-                            },
                             "description": {
                                 "type": "string",
                                 "description": "Description of the input channel"
                             },
-                            "pattern": {
-                                "type": "string",
-                                "description": "Pattern of the input channel, given in Java glob syntax"
-                            },
-                            "default": {
-                                "type": ["string", "number", "boolean", "array", "object"],
-                                "description": "Default value for the input channel"
-                            },
-                            "enum": {
+                            "structure": {
                                 "type": "array",
-                                "description": "List of allowed values for the input channel",
+                                "description": "Structure of the input channel",
                                 "items": {
-                                    "type": ["string", "number", "boolean", "array", "object"]
-                                },
-                                "uniqueItems": true
+                                    "type": "object",
+                                    "structureProperties": {
```

Comment:
I've never seen the `structureProperties` keyword, is this some feature that I don't know about or did you mean `patternProperties` here?

Reply:
my bad, fixed now, thanks!